deepctr_torch/layers/activation.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding:utf-8 -*-
+"""
+
+Author:
+    Yuef Zhang
+
+"""
+import sys
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+device = 'cuda:0' if torch.cuda.is_available() else 'cpu'
+
+class Dice(nn.Module):
+    """
+    
+    References
+        - [Zhou G, Zhu X, Song C, et al. Deep interest network for click-through rate prediction[C]//Proceedings of the 24th ACM SIGKDD International Conference on Knowledge Discovery & Data Mining. ACM, 2018: 1059-1068.](https://arxiv.org/pdf/1706.06978.pdf)
```

Comment:
Do not add another file in `layer`

Reply:
> Do not add another file in `layer`

I put this file here after referred to tensorflow version's file structure. So, where should I add the new activation layer?